docs/setup/quick-start.md

```diff
@@ -1,6 +1,9 @@
-There are some ways to set up Sedna, depends on your use case:
+There are some ways to setup Sedna, depends on youruse case:
 - If you have none Kubernetes environment and don't want to install Kubernetes manually, you can follow [the instruction](./all-in-one.md) to install all-in-one Sedna environment.
-- Else you can follow [this instruction](./install.md) to install Sedna on existing Kubernetes cluster.
-- Also [there is a local up script](./local-up.md) which is mainly used for developing Sedna, it boots a local Kubernetes cluster and installs Sedna based on local Sedna repository.
+- Otherwise, when Kubernetes is ready, you can follow [this instruction](./install.md) to install Sedna on the existing Kubernetes cluster.
+
+- One more thing: we also have a local-up install script available for easier Sedna development. In details, the local-up version boots a local Kubernetes cluster and installs Sedna based on a local repository, you can see in [here](./local-up.md).
+
+
+After setup Sedna, you can try out [examples](/examples).
```

Comment:
setup -> set up

Reply:
Sorry, I have reverted quick-start to the old version by mistake